Fastfile

```diff
@@ -75,115 +73,113 @@ end
 
 # --------- Custom functions --------- #
 
-def _setup()
+def _setup
   ENV['SCAN_SCHEME'] = ENV['GYM_SCHEME']
-  if ENV['SCAN_DEVICE'] == nil
-    ENV['SCAN_DEVICE'] = "iPhone 6 (9.3)"
-  end
-  if is_ci && ENV['TAB_XCODE_PATH'] != nil
-    xcode_select(ENV['TAB_XCODE_PATH'])
-  end
-  if ENV['TAB_UI_TEST_SCHEME'] != nil && ENV['TAB_OUTPUT_TYPES'] == nil
-    ENV['TAB_REPORT_FORMATS'] = "html"
-  end
-  if ENV['TAB_UI_TEST_SCHEME'] != nil && ENV['TAB_UI_TEST_DEVICES'] == nil
-    ENV['TAB_UI_TEST_DEVICES'] = "iPhone 8"
+  ENV['SCAN_DEVICE'] = 'iPhone 6 (9.3)' if ENV['SCAN_DEVICE'].nil?
+  xcode_select(ENV['TAB_XCODE_PATH']) if is_ci && !ENV['TAB_XCODE_PATH'].nil?
+
+  unless ENV['TAB_UI_TEST_SCHEME'].nil?
+    ENV['TAB_REPORT_FORMATS'] = 'html' if ENV['TAB_OUTPUT_TYPES'].nil?
+    ENV['TAB_UI_TEST_DEVICES'] = 'iPhone 8' if ENV['TAB_UI_TEST_DEVICES'].nil?
   end
 end
 
-def _buildAndDeployToHockey()
+def _build_and_deploy_to_hockey
   icon_overlay(version: get_version_number)
-  _set_build_number()
-  _build_ipa()
-  _upload_to_hockey()
+  _set_build_number
+  _build_ipa
+  _upload_to_hockey
 end
 
-def _set_build_number()
-  build_number = "0"
+def _set_build_number
   use_timestamp = ENV['TAB_USE_TIME_FOR_BUILD_NUMBER'] || false
-  if use_timestamp
-    build_number = Time.now.strftime("%y%m%d%H%M")
-  else
-    build_number = ENV['BUILD_NUMBER']
-  end
+  build_number =
+    if use_timestamp
+      Time.now.strftime("%y%m%d%H%M")
+    else
+      ENV['BUILD_NUMBER']
+    end
```

Comment:
Is it possible here to do something like `build_number = use_timestamp ? Time.now.strftime("%y%m%d%H%M") : ENV['BUILD_NUMBER']`?

Reply:
Maybe, it MIGHT be too long in line length though

Reply:
No problem either way, just got those 1 line feels!